diff --git a/crypto/fipsmodule/rand/urandom_test.cc b/crypto/fipsmodule/rand/urandom_test.cc
index 5935aaf..1119778 100644
--- a/crypto/fipsmodule/rand/urandom_test.cc
+++ b/crypto/fipsmodule/rand/urandom_test.cc
@@ -29,6 +29,10 @@
 #include <sys/syscall.h>
 #include <sys/user.h>
 
+#if !defined(PTRACE_O_EXITKILL)
+#define PTRACE_O_EXITKILL (1 << 20)
+#endif
+
 #if defined(OPENSSL_NO_ASM)
 static int have_rdrand() { return 0; }
 #endif
